app/helpers/advice_page_helper.rb

```diff
@@ -182,5 +182,24 @@ def dashboard_alerts_for_group(dashboard_alerts, group)
   def t_weekday(week_day)
     I18n.t('date.day_names')[week_day]
   end
+
+  #sort an array of SchoolPeriod objects
+  def sort_school_periods(periods)
+    periods.sort { |a, b| a.start_date <=> b.start_date }
+  end
+
+  def can_compare_holiday_usage?(holiday, holiday_usage)
+    return false unless holiday_usage.usage.present?
+    return false unless holiday_usage.previous_holiday_usage.present?
+    Time.zone.today > holiday.end_date
+  end
+
+  def within_school_period?(school_period)
+    @analysis_dates.end_date > school_period.start_date && @analysis_dates.end_date < school_period.end_date
+  end
+
+  def average_daily_usage(usage, school_period)
+    return usage.kwh / (school_period.end_date - school_period.start_date).to_i
```

Comment:
return not needed here.  Also, just to double check you didn't mean to use `to_f` here?

Reply:
You're right the to_i is not really needed.